src/_samples/index.md

```diff
@@ -252,26 +268,37 @@ abstract class Describable {
 }
 {% endprettify %}
 
-Any class extending `Describable` has the `describeWithEmphasis()` method, which calls the extender's implementation of `describe()`.
+Any class extending `Describable` or using it as a mixin has the `describeWithEmphasis()` method, which calls the extender's / base class implementation of `describe()`.
```

Comment:
extender's / base class
->
extender or base class

(I usually avoid "/" in text, since it can be ambiguous. https://developers.google.com/style/slashes)

Reply:
(Since the mixin has been dropped, I've reverted to the original sentence.)